src/mcts/search.cc

```diff
@@ -828,8 +828,7 @@ void SearchWorker::DoBackupUpdate() {
 // 7. UpdateCounters()
 //~~~~~~~~~~~~~~~~~~~~
 void SearchWorker::UpdateCounters() {
-  search_
-      ->UpdateRemainingMoves();  // Update remaining moves using smart pruning.
+  search_->UpdateRemainingMoves(); // Uses smart pruning.
```

Comment:
It actually doesn't use smart pruning, but instead computes counters for smart pruning to use.
(in PickNodeToExtend())

Reply:
Well then I would say the original comment was inaccurate too?

Reply:
Kind of. It should have be "for" rather than "using".